web-app/js/unifina/signalPath/specific/chartInput.js

```diff
@@ -22,8 +22,10 @@ SignalPath.ChartInput = function(json, parentDiv, module, type, pub) {
     // Use 1-based index for display to the user
     var displayedAxis = json.yAxis + 1
     // Cycle Y-axis button
-    var $yAxisSelectorButton = $("<div class='y-axis-number btn "+btnDefaultClass+" btn-xs "+popoverClass+" popover-colorful'></div>")
-    
+    var $yAxisSelectorButton = $('<div/>', {
```

Comment:
While manually testing, I noticed that the y-axis selector button is not shown beside the chart input at all! Changes look ok, so not sure if it's related to these changes or some earlier ones?

Reply:
After investigating a bit it seems that this file is never added to the UI (it's not in `ApplicationResources.groovy`)

Reply:
#518 created